[docs][material-ui][joy-ui] Simplify the Quickstart section on the Usage page (#38385)
mui/material-ui: docs/data/material/getting-started/usage/usage.md

## Quickstart

> After [installation](/material-ui/getting-started/installation/), import any Material UI component and start playing around with them.

Reviewer: ```suggestion
After [installation](/material-ui/getting-started/installation/), you can import any Material UI component and start playing around.
```
